frontend/src/components/header.js

```diff
@@ -7,9 +7,28 @@ import { createPortal } from "react-dom";
 import { Button } from "./ui";
 import { navigate } from "@reach/router";
 import { useReset } from "../action-hooks";
+import iconHome from "./../images/home.svg";
 
 function Header({ show, direction }) {
   const current = useCurrentGame();
+  const iconPlane = [];
+  for (let i = 0; i < 7; i++) {
```

Comment:
What you want here instead of a `for` loop is:
```js
const iconPlane = Array.from({ length: 7 }, (_, index) => {
  // return your SVG using index here
});
```

Reply:
Yes! It's a better way! I go to fix it!